target/linux/lantiq/Makefile

```diff
@@ -11,7 +11,7 @@ BOARDNAME:=Lantiq
 FEATURES:=squashfs
 SUBTARGETS:=xrx200 xway xway_legacy falcon ase
 
-KERNEL_PATCHVER:=4.19
+KERNEL_PATCHVER:=5.4
 KERNEL_TESTING_PATCHVER:=5.4
```

Comment:
as `KERNEL_PATCHVER` is set to the same as `KERNEL_TESTING_PATCHVER` you can as well remove the `KERNEL_TESTING_PATCHVER` line.